ml/src/test/scala/ai/h2o/sparkling/ml/features/H2OWord2VecTestSuite.scala

```diff
@@ -75,8 +79,14 @@ class H2OWord2VecTestSuite extends FunSuite with Matchers with SharedH2OTestCont
   test("Basic Word2Vec") {
```

Comment:
I did not change much in the test besides that one which didn't check anything anyway mostly to prove the new impl passes the same tests as the old one